integration_test/third_party_apps_data/test_config.yaml

```diff
@@ -23,3 +23,5 @@ per_application_overrides:
     platforms_to_skip: *common_skips
   # Note: New applications are not supposed to add additional skips
   # here unless deemed absolutely necessary.
+  couchdb: # couchdb is not supported on sles
+    platforms_to_skip: [sles-12, sles-15]
```

Comment:
The `submodules/collectd` and `submodules/fluent-bit` might have sneaked in accidentally. Diff for those 2 submodules is not required for this PR, right?

![image](https://user-images.githubusercontent.com/5287526/152091978-7418231f-54f2-4472-9462-3f9e9adff356.png)

Reply:
fixed

Reply:
Thanks!